src/api/services/lectureService.ts

```
@@ -0,0 +1,486 @@
+    
+    // Extract basic information
+    result.subjectName = getTableCellByHeader('교과목명', '과목명');
+    result.subjectNameEng = getTableCellByHeader('영문교과목명', '영문명');
+    result.courseCode = getTableCellByHeader('학수번호');  // e.g., BBAB67057
+    
+    // Extract and validate courseNumber (ensure 4-digit)
+    const rawCourseNumber = getTableCellByHeader('과목번호');
+    result.courseNumber = rawCourseNumber?.substring(0, 4); // Ensure 4-digit
+    
+    if (isDev && rawCourseNumber) {
+    }
```

ESLint

[error] Empty block statement.

(no-empty)